app/models/audit_trail.rb

```diff
@@ -15,21 +15,51 @@ def changed_fields
     relevant_fields = modified.map do |key, _value|
       key unless IRRELEVANT_FIELDS.include? key
     end
-
     relevant_fields.compact.map(&:humanize)
   end
 
   # TODO: properly render null values like in special circumstances
   # Something like this should work:
   # "HAHA I WIN" if (fields.all?{|f| f.blank? || (f.flatten.all? &:blank? if f.respond_to?('each'))})
+
+  def formatDates(hash)
+    for key in ['appointment_date', 'initial_call_date', 'pledge_generated_at']
+      if hash.has_key? key
+        hash[key] = Date.strptime(hash[key].to_s, "%Y-%m-%d %H:%M:%S %Z").strftime('%m-%d-%Y')
+      end
+    end
+    return hash
+  end
+
+  # TODO make columns more consistent in height
```

Comment:
Yep, def special circumstances is the array of checkboxes. May want to rename this method so it refers to special circumstances specifically, so we don't have to guess.

What's the TODO here refer to? Should we spin up a separate issue for it?